Tasks/Maven/sonarqube-common/restresponse.ts

```diff
@@ -0,0 +1,13 @@
+// Data class representing an HTTPS response from the SonarQube server.
```

Comment:
All files under Tasks/Maven/sonarqube-common/ and Tasks/Gradle/sonarqube-common/ are identical.